src/task13/task13.js

```diff
@@ -1,22 +1,25 @@
-class Vec {
-    constructor(x, y) {
-        this.x = x;
-        this.y = y;
-    }
-    get length() {
-        return (Math.sqrt(Math.pow(this.x, 2) + Math.pow(this.y, 2)));
-    }
-    plus(vector) {
-        return new Vec(this.x + vector.x, this.y + vector.y);
-    }
-    minus(vector) {
-        return new Vec(this.x - vector.x, this.y - vector.y);
-    }
+function Vec(x, y) {
```

Comment:
What functions start with capital letter?

Reply:
what d'you mean
constructors do, don't they?

Reply:
like
function Rabbit(type) {
  this.type = type;
}

Reply:
Exactly